nni/nas/evaluator/functional.py

```diff
@@ -1,14 +1,16 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-import nni
-from .evaluator import Evaluator
+from __future__ import annotations
 
+from nni.common.serializer import SerializableObject
+from .evaluator import MutableEvaluator
 
-@nni.trace
-class FunctionalEvaluator(Evaluator):
+
+class FunctionalEvaluator(MutableEvaluator):
```

Comment:
is the usage of this evaluator different from before, user need manual add `nni.trace`?

Reply:
FunctionalEvaluator implements the traceable interface by itself, so that it doesn't need the `nni.trace` decorator.